agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3678,23 +3679,22 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        auto apconfHeader =
-            cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
-        auto tlvAp = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
-        if (tlvAp == nullptr) {
+        auto c  = cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE);
```

Comment:
When if c == nullptr?
second, I don't think using 'c' as a name is good idea. It is not meaningful. 
Please consider to change it to "cmdu_header".

Reply:
Fixed.
Regarding using short names - generally I agree, but in this case we only use the return pointer to check if `create` succeeded. In such cases I prefer to keep the line short by using a short variable since it is not used other than error checking anyway. I encourage you to do the same, will prevent clang-format from "uglying" the code.

Reply:
`if (!cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE)) {`
same length ~